testsuite/tests/deletion/test.yaml

```diff
@@ -0,0 +1 @@
+description: "minimal"
```

Comment:
the description should be changed to "deletion" (I wish e3-testsuite didn't force us to write these yaml files but that's anoter project for another day)